pkg/agent/openflow/client.go

```diff
@@ -341,10 +342,18 @@ type Client interface {
 	// InstallMulticlusterClassifierFlows installs flows to classify cross-cluster packets.
 	InstallMulticlusterClassifierFlows(tunnelOFPort uint32, isGateway bool) error
 
-	// UninstallMulticlusterFlows removes cross-cluster flows matching the given clusterID on
+	// InstallMulticlusterPodFlows installs flows to handle cross-cluster packets from Multi-cluster Gateway to
+	// regular Nodes.
+	InstallMulticlusterPodFlows(podIP net.IP, tunnelPeerIP net.IP) error
+
+	// UninstallMulticlusterFlows removes cross-cluster flows matching the given cache key on
 	// a regular Node or a Gateway.
 	UninstallMulticlusterFlows(clusterID string) error
 
+	// UninstallMulticlusterPodFlows removes Pod flows matching the given cache key on
+	// a Gateway.
+	UninstallMulticlusterPodFlows(podIP string) error
```

Comment:
Use `net.IP` as other funcs?

Reply:
I feel it's unnecessary to use net.IP here since we already use string type of IP in pod_route_controller.go. If we change it to net.IP, we need to convert string to `net.IP` in pod_route_controller.go, then convert `net.IP` to string again in this `UninstallMulticlusterPodFlows`.